### PR TITLE
Fixed styles in reset password page

### PR DESCRIPTION
**Pull request by Yurenko**
## Description

Change styles in reset password page according to figma. And form is without different moving when appear an error.

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| 203142961-dc0e44c4-f945-4bd9-a650-7322f965bfa4 (1) | Знімок екрана 2022-11-23 100703 |
| 203142969-918c0e90-390a-4279-9f7a-d8764c30135c | Знімок екрана 2022-11-23 100612 |
| Знімок екрана 2022-11-23 103259 | Знімок екрана 2022-11-23 103331 |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by Pipuss — DISMISSED** [review]

**Review by MickhaelSh — DISMISSED** [review]

File: src/components/auth-form/auth-form-heading/auth-form-heading.styles.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ export const useStyles = makeStyles((theme) => ({
   heading: {
     fontFamily: theme.typography.fontFamily,
     color: theme.palette.textColor,
-    fontSize: theme.typography.h2.fontSize,
+    fontSize: '34px',
```
Comment (byavdoshniak) [inline src/components/auth-form/auth-form-heading/auth-form-heading.styles.js:7]: change to theme.typography.h1.fontSize

File: src/pages/recovery/recovery.styles.js [hunk 2]
```diff
@@ -2,50 +2,49 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles((theme) => ({
   emailInput: {
+    '& h4': {
+      fontSize: '34px',
+      lineHeight: '46px'
```
Comment (byavdoshniak) [inline src/pages/recovery/recovery.styles.js:7]: change this to h1

File: src/pages/recovery/recovery.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ const Recovery = () => {
             value={values.email}
             error={!!errorMsg}
           />
-          {errorMsg && <div className={styles.errorText}>{errorMsg}</div>}
+          <div className={styles.errorText}>{errorMsg || ''}</div>
```
Comment (byavdoshniak) [inline src/pages/recovery/recovery.js:71]: use helperText props for TextField component with alternative string with 1 whiteSpace

File: src/pages/recovery/recovery.styles.js [hunk 4]
```diff
@@ -2,50 +2,49 @@ import { makeStyles } from '@material-ui/core/styles';
...
+    },
+    '& label': {
+      fontWeight: 400,
+      fontSize: '16px',
+      transform: 'translate(14px, 12px) scale(1)'
+    },
+    '& div > input': {
+      padding: '11px 14px !important',
```
Comment (byavdoshniak) [inline src/pages/recovery/recovery.styles.js:15]: remove important, use inputProps for textField component or check child classes for input

File: src/pages/recovery/recovery.styles.js [hunk 5]
```diff
@@ -2,50 +2,49 @@ import { makeStyles } from '@material-ui/core/styles';
...
+      position: 'relative',
```
Comment (byavdoshniak) [inline src/pages/recovery/recovery.styles.js:16]: check if relative is needed here

**Review by byavdoshniak — CHANGES_REQUESTED** [review]

File: src/pages/recovery/recovery.styles.js (hunk 2 above)
Comment (Yurenko) [reply src/pages/recovery/recovery.styles.js]: I deleted it. It doesn't need here.

File: src/pages/recovery/recovery.js (hunk 3 above)
Comment (Yurenko) [reply src/pages/recovery/recovery.js]: Fixed.

File: src/pages/recovery/recovery.styles.js (hunk 4 above)
Comment (Yurenko) [reply src/pages/recovery/recovery.styles.js]: Removed important.

File: src/pages/recovery/recovery.styles.js (hunk 5 above)
Comment (Yurenko) [reply src/pages/recovery/recovery.styles.js]: I deleted it. It doesn't need here.

**Review by Pipuss — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Review by IrynaMel — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

**Pull request merged**